packages/rolldown/src/options/output-options.ts

```diff
@@ -111,32 +111,75 @@ export interface OutputOptions {
+       * export default defineConfig({
+       *   advancedChunks: {
+       *     groups: [
+       *       {
+       *         name: 'libs',
+       *         test: /node_modules/,
+       *       },
+       *     ],
+       *   },
+       * });
+       * ```
+       * will create a chunk named `libs-[hash].js` in the end.
        */
       name: string;
       /**
        * - Type: `string | RegExp | ((id: string) => boolean | undefined | void);`
        *
        * Controls which modules are captured in this group.
        *
-       * If `test` is a string, the module whose id contains the string will be captured.
-       * If `test` is a regular expression, the module whose id matches the regular expression will be captured.
-       * If `test` is a function, modules for which `test(id)` returns `true` will be captured.
-       * if `test` is empty, any module will be considered as matched.
+       * - If `test` is a string, the module whose id contains the string will be captured.
+       * - If `test` is a regular expression, the module whose id matches the regular expression will be captured.
+       * - If `test` is a function, modules for which `test(id)` returns `true` will be captured.
+       * - If `test` is empty, any module will be considered as matched.
        */
       test?: StringOrRegExp | ((id: string) => boolean | undefined | void);
       /**
        * - Type: `number`
+       * - Default: `0`
        *
        * Priority of the group. Group with higher priority will be chosen first to match modules and create chunks. When converting the group to a chunk, modules of that group will be removed from other groups.
        *
        * If two groups have the same priority, the group whose index is smaller will be chosen.
+       *
+       * For example,
+       *
+       * ```js
+       * import { defineConfig } from 'rolldown';
+       *
+       * export default defineConfig({
+       *  advancedChunks: {
+       *   groups: [
+       *      {
+       *        name: 'react',
+       *       test: /node_modules\/react/,
+       *       priority: 1,
+       *      },
+       *      {
+       *       name: 'other-libs',
+       *       test: /node_modules/,
+       *      priority: 2,
+       *      },
+       *   ],
+       * });
+       * ```
+       *
+       * This is a clearly __incorrect__ example. Though `react` group is defined before `other-libs`, it has a lower priority, so the modules in `react` group will be captured in `other-libs` group.
        */
       priority?: number;
       /**
        * - Type: `number`
        * - Default: `0`
        *
-       * Minimum size of the desired chunk. If accumulated size of captured modules is smaller than this value, this group will be ignored.
+       * Minimum size of the desired chunk. If the final size of this group is smaller than this value, it will be ignored. Modules in this group will fall back to the `automatic chunking` if they are not captured by any other group.
```

Comment:
```suggestion
       * Minimum size of the desired chunk. If the accumulated size of the captured modules by this group is smaller than this value, it will be ignored. Modules in this group will fall back to the `automatic chunking` if they are not captured by any other group.
```
I think "the accumulated size" is more clear here.